charts/openfaas-cloud/templates/ingress/ingress-wildcard.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: openfaas-ingress
+  namespace: {{ .Values.global.coreNamespace }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/limit-connections: "20"
+    nginx.ingress.kubernetes.io/limit-rpm: "600"
+  labels:
+    app: faas-netesd
+spec:
+  {{- if .Values.tls.enabled }}
+  tls:
+  - hosts:
+    - {{ printf "*.%s" .Values.global.rootDomain | quote }}
```

Comment:
`printf` is a new one on me in helm charts, what's the advantage?

Reply:
if you want to pass the result to `quote`